docs/_writing-tests/reftests.md

```diff
@@ -112,10 +112,52 @@ only guaranteed to be after those events.
 ## Fuzzy Matching
 
 In some situations a test may have subtle differences in rendering
-compared to the reference due to, e.g., anti-aliasing. This may cause
-the test to pass on some platforms but fail on others. In this case
-some affordance for subtle discrepancies is desirable. However no
-mechanism to allow this has yet been standardized.
+compared to the reference due to, e.g., anti-aliasing. To allow for
+these small differences, we allow tests to specify a fuzziness
+characterised by two parameters:
+
+ * A difference in the per-channel color value for any pixel.
+ * A number of total pixels that may be different
+
+To specify the fuzziness in the test file one may add a `<meta
+name=fuzzy>` element (or, in the case of more complex tests, to any
+page containing the `<link rel=[mis]match>` elements). In the simplest
+case this has a `content` attribute containing the parameters above,
+separated by a colon e.g.
+
+```
+<meta name=fuzzy content="maxDifference=15;totalPixels=300">
+```
+
+would allow for a  difference of exactly 15 / 255 on any color channel
+and 300 exactly pixels total difference. The argument names are optional
+and may be elided; the above is the same as:
+
+```
+<meta name=fuzzy content="15;300">
+```
+
+The values may also be given as ranges e.g.
+
+```
+<meta name=fuzzy content="maxDifference=10-15;totalPixels=200-300">
+```
+
+In this case the maximum pixel difference must be in the range
```

Comment:
Also, "the maximum pixel difference" could also use some clarification. Based on our discussion, I think you mean the difference of the colour value of the channel with the largest difference for a pixel. I really struggle to concisely define this without ambiguity, so perhaps it's worthwhile to formally define the terminology at the top of the document, e.g.

Actual(p, c) and Expected(p, c) are the actual and expected colour values of channel c at pixel p. D(p) = max{|Actual(p, c) - Expected(p, c)| for c in (R, G, B)}.